ecl/eclccserver/eclccserver.cpp

```diff
@@ -464,7 +464,30 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
 #ifdef _CONTAINERIZED
         if (!globals->getPropBool("@useChildProcesses", false) && !globals->hasProp("@workunit"))
         {
-            runK8sJob("eclccserver", wuid, wuid, globals->getPropBool("@deleteJobs", true));
+            try
+            {
+                runK8sJob("compile", wuid, wuid, globals->getPropBool("@deleteJobs", true));
+            }
+            catch (IException *E)
+            {
```

Comment:
thinking it would better if the block of code below was handled outside of the catch? (e.g. set and and test after try block)
In case this code throws and causes unhandled exception issues.